### ISSUE TITLE
Show route map in admin PIREP screen

### ISSUE THREAD

**Issue by nabeelio** [phpvms/phpvms#946]

**Comment by arthurpar06** [comment]
Since there is a "View" butotn directly in the admin table that redirects to the frontend where the map is located, I'm not sure if it's necessary. We could add it to the edit pireps page, but wouldn't that overload it?

**Comment by FatihKoz** [comment]
Maybe a small map in the edit page would be ok, but do we really need it ?

A slight change on edit page will be much better, like putting rows in columns and seeing stuff side by side (like fields and log entries) instead of down.

**Comment by BossOfGames** [comment]
If admins can accept/reject pireps on the view side, then I think it's not needed. I don't know if there's any demand for this feature.

**Comment by FatihKoz** [comment]
No, they can not accept/reject from frontend if this is what you are saying.

There was a request for the map at admin side, maybe 3-4 years ago, to ease out manual evaluation etc. After some chat, we added the "View Pirep" button (probably Maco sent the PR) there to simply fulfill that need. But this remained open here.

If admins really need the map to check, then yes a small map can be placed here, as all the data is already present. But when I need to check the route flown, I simply click on the view pirep link and inspect it there, then decide what to do. Which happens rare, maybe once a month maybe less.

Mostly when something strange happens, like a faulty airport detection and diversion (close proximity to another airport), or a crash during takeoff (which is again logged in as diversion). I check the map to understand the flight path and fix properly. For daily normal ops, it is not needed at all (if you are not a freak checking every single pirep for sid/star and directs etc). 😉

**Comment by FatihKoz** [comment]
Something like this maybe

![image](https://github.com/nabeelio/phpvms/assets/74361521/410d671a-74c3-4275-b79a-50cb4c79ef2c)
